confignode/src/main/java/org/apache/iotdb/confignode/manager/load/balancer/RouteBalancer.java

```diff
@@ -219,8 +234,8 @@ public void startRouteBalancingService() {
                 leaderBalancingExecutor,
                 this::balancingRegionLeader,
```

Comment:
Is there adding a config, can close balance?

Reply:
Yes, I've added